src/common/BaseAnnotator.ts

```diff
@@ -97,8 +101,16 @@ export default class BaseAnnotator extends EventEmitter {
             this.containerEl.classList.remove(CSS_CONTAINER_CLASS);
         }
 
-        if (this.annotatedEl) {
-            this.annotatedEl.classList.remove(CSS_LOADED_CLASS);
+        const annotatedElement = this.annotatedEl;
+
+        if (annotatedElement) {
+            annotatedElement.classList.remove(CSS_LOADED_CLASS);
+
+            Object.values(ANNOTATION_CLASSES).forEach(className => {
```

Comment:
Could this logic be pulled into a method like `removeAnnotationClasses` since it's used here and also in `DocumentAnnotator`?

Reply:
Yep that's a good idea